python/ray/tune/tune.py

```diff
@@ -14,18 +14,10 @@
 from ray.tune.syncer import wait_for_sync, set_sync_periods, SyncConfig
 from ray.tune.trial_runner import TrialRunner
 from ray.tune.progress_reporter import CLIReporter, JupyterNotebookReporter
-from ray.tune.schedulers import (HyperBandScheduler, AsyncHyperBandScheduler,
-                                 FIFOScheduler, MedianStoppingRule)
+from ray.tune.schedulers import _SCHEDULERS
```

Comment:
hmm actually maybe we should just replace this with `create_scheduler`, since it basically does the same thing?

Reply:
`from ray.tune.schedulers import create_scheduler`, and then we can get rid of `_SCHEDULERS` all together.

Reply:
that's what I was thinking when I saw this issue—was going to ask you after thinking about this :)

but yes, I'm pretty confident we can safely get rid of `_SCHEDULERS` and instead use `create_scheduler` so I'm in favor of using create_scheduler. If RLLib really needs the dictionary we can make the dictionaries public like `_SCHEDULERS`

Reply:
Sure, that seems better!

Reply:
Is it necessary to have closures inside `create_scheduler` to import all the schedulers, when they are already imported? I will also add the fifo and pbt replay schedulers to `create_scheduler` since they are currently left out.

Reply:
> Is it necessary to have closures inside `create_scheduler` to import all the schedulers, when they are already imported?

hmm I'm not sure what you mean; can you elaborate?

> I will also add the fifo and pbt replay schedulers to `create_scheduler` since they are currently left out.

Yes! thanks :)

Reply:
`create_scheduler` has a few functions to import and return the schedulers, like `_import_async_hyperband_scheduler` but all of the schedulers are already imported at the top anyway. Seems redundant to add functions to import them again.

Reply:
ohhh haha yeah you're right we don't need the closures; we can just make `SCHEDULER_IMPORT` a dict of `{str: tune.schedulers.TrialScheduler}`. @richardliaw should we do this in another PR or make the changes here?

Reply:
Awesome, will do :+1:

Reply:
I see now why fifo and pbt replay were left out from create_schedulers: they don't take the same args (`metric` and `mode` in particular.) Shall I remove these parameters from `create_scheduler` and just leave the `**kwargs` so that it can create these 2 schedulers (since fifo is meant to be default)?

Reply:
yep!